src/toil/common.py

```diff
@@ -674,6 +646,12 @@ def createBatchSystem(config, jobStore=None, userScript=None):
 
         return batchSystemClass(**kwargs)
 
+    @classmethod
+    def clean(cls, jobStoreLocator):
+        from toil.jobStores.abstractJobStore import AbstractJobStore
+        locator = AbstractJobStore.Locator.parse(jobStoreLocator)
+        locator.outer.cleanJobStore(locator)
```

Comment:
This doesn't read well. Can you rename outer to jobStore? The `.outer` idiom only reads well within a class, not in code outside of the class due to lack of context.